### ISSUE TITLE
Grassblock variants in new model

### ISSUE THREAD

**Issue by InrcedibleHolg** [InrcedibleHolg/HerrSommer-1.13#4]
blockstates/grass.json -->  blockstates/grass_block.json
Varianten in den Texturen
grass_top10.png  grass_top13.png  grass_top16.png  grass_top19.png  grass_top3.png  grass_top6.png  grass_top9.png
grass_top11.png  grass_top14.png  grass_top17.png  grass_top1.png   grass_top4.png  grass_top7.png  grass_top.png
grass_top12.png  grass_top15.png  grass_top18.png  grass_top2.png   grass_top5.png  grass_top8.png

**Comment by InrcedibleHolg** [comment]
* grass_block.json default 
```json
{
    "variants": {
        "snowy=false": [
            { "model": "block/grass_block" },
            { "model": "block/grass_block", "y": 90 },
            { "model": "block/grass_block", "y": 180 },
            { "model": "block/grass_block", "y": 270 }
        ],
        "snowy=true":  { "model": "block/grass_block_snow" }
    }
}
```

**Comment by InrcedibleHolg** [comment]
* grass_block.json model

```json
{   "parent": "block/block",
    "textures": {
        "particle": "block/dirt",
        "bottom": "block/dirt",
        "top": "block/grass_block_top",
        "side": "block/grass_block_side",
        "overlay": "block/grass_block_side_overlay"
    },
    "elements": [
        {   "from": [ 0, 0, 0 ],
            "to": [ 16, 16, 16 ],
            "faces": {
                "down":  { "uv": [ 0, 0, 16, 16 ], "texture": "#bottom", "cullface": "down" },
                "up":    { "uv": [ 0, 0, 16, 16 ], "texture": "#top",    "cullface": "up", "tintindex": 0 },
                "north": { "uv": [ 0, 0, 16, 16 ], "texture": "#side",   "cullface": "north" },
                "south": { "uv": [ 0, 0, 16, 16 ], "texture": "#side",   "cullface": "south" },
                "west":  { "uv": [ 0, 0, 16, 16 ], "texture": "#side",   "cullface": "west" },
                "east":  { "uv": [ 0, 0, 16, 16 ], "texture": "#side",   "cullface": "east" }
            }
        },
        {   "from": [ 0, 0, 0 ],
            "to": [ 16, 16, 16 ],
            "faces": {
                "north": { "uv": [ 0, 0, 16, 16 ], "texture": "#overlay", "tintindex": 0, "cullface": "north" },
                "south": { "uv": [ 0, 0, 16, 16 ], "texture": "#overlay", "tintindex": 0, "cullface": "south" },
                "west":  { "uv": [ 0, 0, 16, 16 ], "texture": "#overlay", "tintindex": 0, "cullface": "west" },
                "east":  { "uv": [ 0, 0, 16, 16 ], "texture": "#overlay", "tintindex": 0, "cullface": "east" }
            }
        }
    ]
}
```

**Comment by InrcedibleHolg** [comment]
* Blockstate File  in HerrSommer  grass.json anruft Name ist grass_block 

```json
{
    "variants": {
        "snowy=false": [
            { "model": "grass_normal", "weight": 100 },
                        { "model": "grass/grass_normal1", "weight": 100 },
                        { "model": "grass/grass_normal2", "weight": 200 },
                        { "model": "grass/grass_normal3", "weight": 20 },
                        { "model": "grass/grass_normal4", "weight": 200 },
                        { "model": "grass/grass_normal5", "weight": 10 },
                        { "model": "grass/grass_normal6", "weight": 5 },
                        { "model": "grass/grass_normal7", "weight": 2 },
                        { "model": "grass/grass_normal8", "weight": 3 },
                        { "model": "grass/grass_normal9", "weight": 10 },
                        { "model": "grass/grass_normal10", "weight": 4 },
                        { "model": "grass/grass_normal11", "weight": 4 },
                        { "model": "grass/grass_normal12", "weight": 4 },
                        { "model": "grass/grass_normal13", "weight": 10 },
                        { "model": "grass/grass_normal14", "weight": 10 },
                        { "model": "grass/grass_normal15", "weight": 10 },
                        { "model": "grass/grass_normal16", "weight": 100 },
                        { "model": "grass/grass_normal17", "weight": 10 },
                        { "model": "grass/grass_normal18", "weight": 20 },
                        { "model": "grass/grass_normal19", "weight": 20 }
        ],
        "snowy=true":  { "model": "grass_snowed" }
    }
}
```

**Comment by InrcedibleHolg** [comment]
User a template  model and different Texturen in final models,  refedencing template as parent

**Comment by InrcedibleHolg** [comment]
start with template file

**Comment by InrcedibleHolg** [comment]
solved with 5ec704f7219d67ac223f5876bfb4f4ad67549962
